src/components/molecules/post/index.tsx

```diff
@@ -1,12 +1,26 @@
 import { RouterOutputs } from "~/utils/api"
 import AtomsAvatar from "../../atoms/avatar"
 import { relTime } from "~/utils/daysjs"
-
+import { MdDeleteForever } from "react-icons/md"
+import { useUser } from "@clerk/nextjs" 
+import { api } from "~/utils/api"
 
 // 上記のtypeと同じですが、APIから取得してるデータで作成みたい。
 type MoleculesPostType = RouterOutputs["posts"]["index"][number];
 
 const MoleculesPost = ({post, author}: MoleculesPostType) => {
+  const { user } = useUser();
+
+  const fetchPosts = api.posts.index.useQuery();
```

Comment:
```suggestion
  const { data: posts }  = api.posts.index.useQuery();
```
この書き方をすることで変数が扱いやすくなりそうです！

before
<img width="526" alt="image" src="https://github.com/maria-skygrid/t3-intro/assets/100871461/2b149f88-c379-423b-b7d5-cf3db179b48e">

after
<img width="696" alt="image" src="https://github.com/maria-skygrid/t3-intro/assets/100871461/ba3260e3-fb2d-4001-83b2-d1b1cecc14cf">

Reply:
@qdi0 
ご確認ありがとうございます！
データを再度習得できるためにこのpostsを定義しました。ただ、refetchを使いますので、これを消します〜